src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php

```diff
@@ -325,6 +339,7 @@ public function configureOptions(OptionsResolver $resolver)
 
         $resolver->setAllowedTypes('choice_list', array('null', 'Symfony\Component\Form\ChoiceList\ChoiceListInterface'));
         $resolver->setAllowedTypes('choices', array('null', 'array', '\Traversable'));
+        $resolver->setAllowedTypes('choice_translation_domain', array('null', 'bool', 'string'));
```

Comment:
maybe the easiest would be to add a normalizer that return `null` when it is `true` so they mean the same thing.